lib/summarize/summarizeAttachment.js

```diff
@@ -48,6 +47,8 @@ function summarizeAttachment (attachment, req, sbol, remote, graphUri) {
       attachment.format.toString().indexOf('image') >= 0
   }
 
+  var summarizeTopLevel = require('./summarizeTopLevel')
```

Comment:
Why not do the require at the top so it’s clear it’s an external dependency?

Reply:
This was a strange one.  I've noticed with recursion that these require variables can get messed up.  The only solution I have found has been to reset them in the body of the code.